xds/build.gradle

```diff
@@ -89,7 +89,7 @@ shadowJar {
     dependencies {
         include(project(':grpc-xds'))
     }
-    relocate 'com.github.googleapis', 'io.grpc.xds.shaded.com.github.googleapis'
+    relocate 'com.google.api', 'io.grpc.xds.shaded.com.google.api'
```

Comment:
Can not relocate `com.google.api`. There are some generated code liek CdsProto.java referencing `com.google.api.AnnotationsProto`.

Reply:
I did shadow wrong for `com.google.api` previously, because I wasn't able to import classes from `googleapis`.

It should be ```relocate 'com.google.api', 'io.grpc.xds.shaded.com.google.api'``` instead of ```relocate 'com.github.googleapis', 'io.grpc.xds.shaded.com.github.googleapis'```

I have verified that importing from `googleapis` works now.

Reply:
You might have to relocate `com.google.api.expr` rather than `com.google.api`.

Reply:
Ok, I'll do that. Thanks for the clarification!